arch/PowerPC/PPCGenRegisterInfo.inc

```diff
@@ -372,33 +372,32 @@ enum {
   PPC_F4RCRegClassID = 6,
   PPC_CRRCRegClassID = 7,
   PPC_CARRYRCRegClassID = 8,
-  PPC_CRRC0RegClassID = 9,
```

Comment:
`PPC_CRRC0RegClassID` got removed, most likely due to LLVM TableGen version mismatch.

Reply:
Also causing build warnings